sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/DotCoverReportParser.java

```diff
@@ -32,10 +32,14 @@
 public class DotCoverReportParser implements CoverageParser {
 
   private static final String TITLE_START = "<title>";
+  // the pattern for the information about a sequence point - (lineStart, columnStart, lineEnd, columnEnd, hits)
+  private static final Pattern SEQUENCE_POINT_PATTERN = Pattern.compile("\\[(\\d++),\\d++,(\\d++),\\d++,(\\d++)]");
```

Comment:
I realize now that we don't really use the 2nd captured number - it's lineEnd but we don't use it